packages/SwingSet/test/virtualObjects/vat-vom-gc-bootstrap.js

```diff
@@ -0,0 +1,30 @@
+import { E } from '@agoric/eventual-send';
+
+export function buildRootObject(_vatPowers) {
+  let other;
+  let bob;
+  let me;
+  let goCount = 3;
+  return harden({
+    async bootstrap(vats) {
+      me = vats.bootstrap;
+      bob = vats.bob;
+      E(bob).prepare();
+      await E(me).go();
+    },
+    go() {
+      if (goCount > 0) {
+        E(bob).getThing(me);
+      } else {
+        E(bob).finish();
+      }
+      goCount -= 1;
+    },
+    deliverThing(thing) {
+      // eslint thinks 'other' is unused, but eslint is wrong.
+      // eslint-disable-next-line no-unused-vars
+      other = thing;
+      E(me).go();
```

Comment:
any particular reason to use ping-ponging messages rather than:?

```js
async bootstrap(vats) {
  await E(vats.bob).prepare();
  for (let i=0; i< 3; i++) {
    other = await E(vats.bob).getThing();
  }
  await E(vats.bot).finish();
}
```

I can imagine the scoping rules around `other` might interfere with GC, I've certainly seen something strange happening on Node.js, but this should be running under XS.

Reply:
That's the way I originally wrote it.  This is a workaround for the problem with promise values not being GC'd.